lisp/ess-developer.el

```diff
@@ -30,130 +30,248 @@
 
 ;;; Code:
 
-;; (require 'ess-site) ;; need to assigne the keys in the map
-
 (defgroup ess-developer nil
   "ESS: developer."
   :group 'ess
   :prefix "ess-developer-")
 
-(defface ess-developer-indicator-face
```

Comment:
I am placing this stuff back. We really need a mode line indicator of the current environment.